lib/client-assets.php

```diff
@@ -535,6 +535,23 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'editor', 'window.wp.oldEditor = window.wp.editor;', 'after'
 	);
 
+	// Used to resize the legacy metabox iframes.
+	wp_enqueue_script(
+		'metabox-resizing-gutenberg',
+		gutenberg_url( 'assets/js/iframeResizer.js' ),
+		array( 'jquery' ),
+		filemtime( gutenberg_dir_path() . 'assets/js/iframeResizer.js' ),
```

Comment:
Since this is an external script, should we use the version number?

Reply:
Yeah, definitely, this will probably be a throwaway script though as aduth pointed out there are already iframe resizing things being done in Gutenberg.